modules/nw-ovn-kubernetes-rollback.adoc

```diff
@@ -31,41 +31,63 @@ endif::[]
 
 . Stop all of the machine configuration pools managed by the Machine Config Operator (MCO):
 
-** Stop the master configuration pool:
+** Stop the `master` configuration pool:
 +
-[source,terminal]
+[source,terminal,subs="+quotes"]
 ----
 $ oc patch MachineConfigPool master --type='merge' --patch \
   '{ "spec": { "paused": true } }'
 ----
 
-** Stop the worker machine configuration pool:
+** Stop the `worker` machine configuration pool:
 +
-[source,terminal]
+[source,terminal,subs="+quotes"]
 ----
 $ oc patch MachineConfigPool worker --type='merge' --patch \
   '{ "spec":{ "paused": true } }'
 ----
 
 . To prepare for the migration, set the migration field to `null` by entering the following command:
 +
-[source,terminal]
+[source,terminal,subs="+quotes"]
 ----
 $ oc patch Network.operator.openshift.io cluster --type='merge' \
   --patch '{ "spec": { "migration": null } }'
 ----
 
-. To start the migration, set the network plugin back to OpenShift SDN by entering the following commands:
+. Apply the patch to the `Network.operator.openshift.io` object to set the network plugin back to OpenShift SDN by entering the following command:
 +
-[source,terminal]
+[source,terminal,subs="+quotes"]
 ----
 $ oc patch Network.operator.openshift.io cluster --type='merge' \
   --patch '{ "spec": { "migration": { "networkType": "OpenShiftSDN" } } }'
+----
++
+[IMPORTANT]
+====
+If you applied the patch to the `Network.config.openshift.io` object before the patch operation finalizes on the `Network.operator.openshift.io` object, the Cluster Network Operator enters into a degradation state. To revert this configuration, you can enter the following command and, if necessary, remove the `applied-cluster` `ConfigMap` object from the `openshift-network-operator` namespace.
 
+[source,terminal]
+----
+$ oc edit Network.operator.openshift.io cluster
+----
+====
+
+. Apply the patch to the `Network.config.openshift.io` object to set the network plugin back to OpenShift SDN by entering the following command:
++
+[source,terminal,subs="+quotes"]
+----
 $ oc patch Network.config.openshift.io cluster --type='merge' \
   --patch '{ "spec": { "networkType": "OpenShiftSDN" } }'
 ----
 
+. Confirm that the migration status of the network plugin for the `Network.config.openshift.io cluster` object is `OpenShiftSDN` by entering the following command:
```

Comment:
This step shall be executed before `$ oc patch Network.config.openshift.io cluster --type='merge' \
  --patch '{ "spec": { "networkType": "OpenShiftSDN" } }'`

Reply:
Moved step.